core/tests/webdriverio/accessibility.js

```diff
@@ -76,70 +62,62 @@ describe('screenreader and keyboard user accessibility features', function() {
   var checkActionShortcuts = async function(key, elementToFocus) {
     await waitFor.presenceOf(elementToFocus, 'Element took too long to load');
     // Should move the focus to the elementToFocus.
-    await browser.actions().sendKeys(key).perform();
-    expect(
-      await browser.driver.switchTo().activeElement()
-        .getAttribute('class')).toEqual(
-      await elementToFocus.getAttribute('class'));
+    await browser.keys(key);
```

Comment:
I just had one question for `browser.keys` and that is how it is determining which is the active element?

Reply:
We are not giving input here, instead we are testing the keyboard shortcuts. It means we are just pressing the keyboard shortcuts and then verifying that the correct element is being selected.